app/views/users/show.html.erb

```diff
@@ -35,4 +35,9 @@
 
   <%= link_to 'Edit', edit_user_path(@user) %> |
   <%= link_to 'Back', users_path %>
+
+  <hr>
+
+  <%= render partial: "partials/user_pledges" %>
```

Comment:
Have you thought about explicitly passing in the pledges (instead of relying on instance variables)? You can see this done in views that require a form (ex. `app/views/wishlist_items/edit.html.erb` uses `<%= render 'form', wishlist_item: @wishlist_item %>` on line 7.)

That technique allows us to reuse the partials more easily because the requirements are explicit.

As a bonus point, that solves one of the failing test cases. 😄 Relying on instance variables can be finicky, but explicitly passed dependencies tend to work as expected.

Another bonus point: it makes it easier to avoid setting the extra ivar. A `<%= render 'partials/user_pledges', pledges: @user.pledges %>` is just as easy to read, and it means we're not passing as much data between the controllers and views!